cernopendata/modules/fixtures/data/docs/cms-getting-started-2011/cms-getting-started-2011.md

```diff
@@ -20,6 +21,12 @@ $ cd CMSSW_5_3_32/src/
 $ cmsenv
 ```
 
+## <a name="container">"Do I have to use a virtual machine?"</a>
+
+Not necessarily. You can also try to analyze CMS data in a Docker container instead of in a virtual machine. If you are interested in that,
```

Comment:
```suggestion
Not necessarily. You can also try to analyse CMS data in a Docker container instead of in a virtual machine. If you are interested in that,
```